roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/ruby
+username = "admin"
+password = "changeme"
```

Comment:
Does username/password need to be configurable?  Or can we assume that it will always be these values?